Resources/Prototypes/DeltaV/Entities/Mobs/Player/silicon.yml

```
@@ -2,14 +2,16 @@
 - type: entity
   id: PlayerBorgMedical
   parent: BorgChassisMedical
-  suffix: Diagnosis, Treatment
+  suffix: Full Suite
   components:
   - type: ContainerFill
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)